recipes/rise/build.sh

```diff
@@ -0,0 +1,2 @@
+"${PYTHON}" setup.py install
```

Comment:
This is not for now... but in the future we should think if this is the proper incantation, check https://github.com/conda-forge/staged-recipes/issues/528 for a long discussion about that...